check.py

```diff
@@ -107,6 +107,10 @@
             cmd += ['-O0'] # test that -O0 does nothing
         else:
           cmd += ['-O']
+        if 'PRINT_FULL' in asm:
```

Comment:
If this is adding a class of tests that can be ran with `BINARYEN_PRINT_FULL` set, then the test that this PR adds should have a name besides just `PRINT_FULL.asm`, something like `PRINT_FULL.unreachable_function_block_type.asm.js` I'd think